materials/emma.scss

```diff
@@ -33,4 +33,8 @@ $code-block-font-size: 0.80em;
 .reveal .slide blockquote {
   border-left: 3px solid $text-muted;
   padding-left: 1.5em;
-}
\ No newline at end of file
+}
+
+.reveal .slide div.sourceCode {
+  margin-bottom: 20px;
+}
```

Comment:
@3mmaRand - just to give you a heads-up on this wee bit of css. 

If it causes you grief, I'll find another way.

Reply:
It's all good!